hs-bindgen/src-internal/HsBindgen/Frontend/AST/Finalize.hs

```diff
@@ -215,15 +215,24 @@ instance Finalize Int.Function where
 
   finalize function = Ext.Function{
         functionArgs = map finalize functionArgs
+      , functionAttrs = map finalizeFunctionAttr functionAttrs
       , functionRes  = finalize functionRes
       }
     where
       Int.Function {
           functionArgs
         , functionRes
+        , functionAttrs
         , functionAnn = NoAnn
         } = function
 
+      -- TODO: for now, @Int.FunctionAttr@ and @Ext.FunctionAttr@ are the exact
+      -- same types, but defined twice. Maybe that will change, see the TODO on
+      -- @Int.FunctionAttr@. If it does not change, maybe we should define a
+      -- common @FunctionAttr@ type in a shared module?
```

Comment:
It doesn't have to be a new module, `External` pulls together definitions from lots of places. Just re-exporting from `Internal` is fine.